projects/1.19/assets/sweetmagic/sweetmagic/lang/zh_cn.json

```
@@ -0,0 +1,1097 @@
+	"item.sweetmagic.endermage_spawn_egg": "法师末影人刷怪蛋",
+	"item.sweetmagic.windwitch_spawn_egg": "风魔女刷怪蛋",
+	"item.sweetmagic.pane2_door_i": "吕字门",
+	"item.sweetmagic.pane4_door_i": "五福门",
+	"item.sweetmagic.elegant_door_i": "花窗门",
+	"item.sweetmagic.arch_door_i": "拱窗门",
+	"item.sweetmagic.arch_plant_door_i": "饰植拱窗门",
+	"item.sweetmagic.simple_door_i": "简易门",
+	"item.sweetmagic.simple_net_door_i": "镶格门",
+	"item.sweetmagic.magic_advance": "高阶射击",
+	"item.sweetmagic.warrior_bracelet": "勇士手镯",
+	"item.sweetmagic.blood_sucking_ring": "吸血指环",
+	"item.sweetmagic.emelald_pias": "绿宝石耳环",
+	"item.sweetmagic.veil_darkness": "夜幕面纱",
+	"item.sweetmagic.pendulum_necklace": "水晶灵摆",
+	"item.sweetmagic.mysterious_fork": "神秘餐叉",
+	"item.sweetmagic.extension_ring": "延伸指环",
+	"item.sweetmagic.vannila_pods": "香草荚",
+	"item.sweetmagic.pineapple_seed": "菠萝种子",
+	"item.sweetmagic.greenpepper_seed": "青椒种子",
+	"item.sweetmagic.spinach_seed": "菠菜种子",
+	"item.sweetmagic.j_radish_seed": "白萝卜种子",
+	"item.sweetmagic.whitenet_seed": "竹荪孢子",
+	"item.sweetmagic.azuki_seed": "红豆种子",
+	"item.sweetmagic.blueberry_seed": "蓝莓",
+	"item.sweetmagic.pineapple": "菠萝",
+	"item.sweetmagic.whitenet": "竹荪",
+	"item.sweetmagic.greenpepper": "青椒",
+	"item.sweetmagic.spinach": "菠菜",
+	"item.sweetmagic.j_radish": "白萝卜",
+	"item.sweetmagic.vannila_essence": "香草精",
+	"item.sweetmagic.vinegar": "醋",
+	"item.sweetmagic.mayonnaise": "蛋黄酱",
+	"item.sweetmagic.salad_dressing": "沙拉酱",
+	"item.sweetmagic.soy_sauce": "酱油",
+	"item.sweetmagic.breadcrumbs": "面包糠",
+	"item.sweetmagic.seaweed": "紫菜",
+	"item.sweetmagic.bonito_flakes": "木鱼花",
+	"item.sweetmagic.dry_seaweed": "干紫菜",
+	"item.sweetmagic.sweet_potato": "奶油红薯",
+	"item.sweetmagic.itigo_daihuku": "草莓大福",
+	"item.sweetmagic.peppers_stuffed_with_meat": "甜椒酿肉",
+	"item.sweetmagic.ogura_toast": "小仓吐司",
+	"item.sweetmagic.choko_cornet": "巧克力牛角包",
+	"item.sweetmagic.sandwich": "三明治",
+	"item.sweetmagic.sandwich_fruit": "水果三明治",
+	"item.sweetmagic.hamburger": "汉堡包",
+	"item.sweetmagic.riceball_salt": "盐饭团",
+	"item.sweetmagic.riceball_salmon": "鲑鱼饭团",
+	"item.sweetmagic.riceball_grilled": "烤饭团",
+	"item.sweetmagic.butadon": "猪排盖饭",
+	"item.sweetmagic.gyuudon": "牛肉盖饭",
+	"item.sweetmagic.kurigohan": "栗子饭",
+	"item.sweetmagic.fried_rice": "炒饭",
+	"item.sweetmagic.sushi_egg": "玉子寿司",
+	"item.sweetmagic.sushi_salmon": "鲑鱼寿司",
+	"item.sweetmagic.salmondon": "三文鱼盖饭",
+	"item.sweetmagic.nikujaga": "土豆炖肉",
+	"item.sweetmagic.salad_mixoil": "炒蔬菜",
+	"item.sweetmagic.salad_mixcorn": "玉米沙拉",
+	"item.sweetmagic.salad_coleslaw": "蔬菜沙拉",
+	"item.sweetmagic.salad_potate": "沙拉土豆",
+	"item.sweetmagic.salad_ohitasi": "凉拌菠菜",
+	"item.sweetmagic.salad_caesar": "凯撒沙拉",
+	"item.sweetmagic.peppers_with_soy_sauce_and_bonito": "炒青椒",
+	"item.sweetmagic.salad_fruit": "水果沙拉",
+	"item.sweetmagic.salmon_carpaccio": "三文鱼排",
+	"item.sweetmagic.spinach_egg": "菠菜蛋羹",
+	"item.sweetmagic.buridaikon": "萝卜干",
+	"item.sweetmagic.salmon_meuniere": "炸三文鱼",
+	"item.sweetmagic.saba_miso": "味噌鱼",
```

原文`鯖味噌煮`
`鯖` 指 鲭科（[Scombridae](https://en.wikipedia.org/wiki/Scombridae)）鱼类的统称，其中包括鲅鱼、金枪鱼等，可统称`鲭鱼`
```suggestion
	"item.sweetmagic.saba_miso": "味噌炖鲭鱼",
```